### PR TITLE
Add 2019-05-13-monitoring-EMS-blog.md

### PR DESCRIPTION
**Pull request by jkld11**

### CODE REVIEW

File: content/blog/2019-05-13-monitoring-EMS-blog.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/6oiQBHTvm6U" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+These operations can be, quite literally, a matter of life or death, so observability is critical. To make the information more accessible, Wetherall said, “we’re trying to ingrain Grafana in the customers that we’re working with, get it exposed, and get everything visual.”
+
+Trapeze’s data environment is based around SQL Server. Because Trapeze’s customers operate in large geographical areas where there isn’t always high-speed cellular coverage -- “I’m talking like 9KB a second,” he said -- Wetherall’s team has in the past year done some work with the spatial side of Postgres, building custom protocols.
```
Comment (tomwilkie) [inline content/blog/2019-05-13-monitoring-EMS-blog.md]: SQL Server implies Microsoft, and you later say Postgres.  Its turns out its both, so you probably want to say that:

"Trapeze’s data environment is based around SQL Server and Postgres."

File: content/blog/2019-05-13-monitoring-EMS-blog.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+The data covers location, connectivity, safety, communication, awareness, and integration. There’s information about where the equipment is and where the people are, and communication with the hospitals. There are many integrated data sources around vehicle telematics and the internet of lifesaving things, such as defibrillators and specialist equipment, as well as those around smart cities, like traffic lights. “We’re monitoring all this in real time, making sure they have connectivity as we do things like run on multiple networks,” said Wetherall. “If one service is out, we need to use others and keep retrying.”
+
+![EMS Slide 17](/assets/img/blog/EMS_17.png)
```
Comment (tomwilkie) [inline content/blog/2019-05-13-monitoring-EMS-blog.md]: Did you add this image?

File: content/blog/2019-05-13-monitoring-EMS-blog.md (hunk 1 above)
Comment (jkld11) [reply content/blog/2019-05-13-monitoring-EMS-blog.md]: Thanks! Will fix.

File: content/blog/2019-05-13-monitoring-EMS-blog.md (hunk 2 above)
Comment (jkld11) [reply content/blog/2019-05-13-monitoring-EMS-blog.md]: I did... Let me see if it renders in staging.

**Pull request merged**